app/templates/auth/change-password.html

```diff
@@ -38,60 +33,37 @@
     <div class="grid-row">
         <div class="column-two-thirds">
             {{ form.hidden_tag() }}
-
-
-            {% if form.old_password.errors %}
-                <div class="validation-wrapper">
-            {% endif %}
-                <div class="question" id="{{ form.old_password.name }}">
-                    {{ form.old_password.label(class="question-heading") }}
-                    {% if form.old_password.errors %}
-                    <p class="validation-message" id="error-old-password-textbox">
-                        {% for error in form.old_password.errors %}{{ error }}{% endfor %}
-                    </p>
-                    {% endif %}
-                    {{ form.old_password(class="text-box", autocomplete="off") }}
-                </div>
-            {% if form.old_password.errors %}
-                </div>
-            {% endif %}
-
-
-            {% if form.password.errors %}
-                <div class="validation-wrapper">
-            {% endif %}
-                <div class="question" id="{{ form.password.name }}">
-                    {{ form.password.label(class="question-heading") }}
-                    <p class="hint">
-                      Must be between 10 and 50 characters
-                    </p>
-                    {% if form.password.errors %}
-                    <p class="validation-message" id="error-password-textbox">
-                        {% for error in form.password.errors %}{{ error }}{% endfor %}
-                    </p>
-                    {% endif %}
-                    {{ form.password(class="text-box", autocomplete="off") }}
-                </div>
-            {% if form.password.errors %}
-                </div>
-            {% endif %}
-
-            {% if form.confirm_password.errors %}
-                <div class="validation-wrapper">
-            {% endif %}
-                <div class="question" id="{{ form.confirm_password.name }}">
-                    {{ form.confirm_password.label(class="question-heading") }}
-
-                    {% if form.confirm_password.errors %}
-                    <p class="validation-message" id="error-confirm-password-textbox">
-                        {% for error in form.confirm_password.errors %}{{ error }}{% endfor %}
-                    </p>
-                    {% endif %}
-                    {{ form.confirm_password(class="text-box", autocomplete="off") }}
-                </div>
-            {% if form.confirm_password.errors %}
-                </div>
-            {% endif %}
+          
+            {% with question = form.old_password.label,
+                    name = "old_password",
+                    error = errors.get('old_password', {}).get('message'),
+                    value = form.old_password.data,
+                    type = "password",
+                    autocomplete = "off"
```

Comment:
Do we need to state `autocomplete = "off"` if this is going to be the default for password type fields?

Reply:
It will be the default, but I also felt I wanted to be explicit in these forms.